src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,54 @@ title = "Releases"
+* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS). This ensures the
+prerequisites for installing Submariner on RHOS are satisfied.
+* New Globalnet metrics were added:
+  * Count of global Egress IPs allocated for Pods/Services per CIDR.
+  * Count of global Egress IPs allocated for clusters per CIDR.
+  * Count of global Ingress IPs allocated for Pods/Services per CIDR.
+* Added support for Globalnet objects verification using subctl diagnose command.
+* Added support for `--broker-namespace` flag while deploying the Broker.
+* Added support for running `subctl diagnose` on single node clusters.
+* Added support for running `subctl diagnose` from a pod in cluster.
+* `subctl cloud prepare` now deploys a dedicated gateway node as a default option on GCP and OpenStack platforms.
+* `subctl show` command now shows information about the Broker CR in the cluster.
+* `subctl gather` now collects the necessary globalnet objects like clusterGlobalEgressIPs, globalEgressIPs, globalIngressIPs.
+* `subctl diagnose` throws a warning when a generic CNI network plugin is detected.
+* New Service Discovery metric was added:
+  * Count of DNS queries handled by Lighthouse plugin.
+
+**Bug fixes**
+
+* Calico is now correctly detected when used as a third-party plugin in OpenShift.
+* Services without selectors can now be resolved across the ClusterSet.
+* `subctl diagnose firewall inter-cluster` command will now work correctly for vxlan cable driver.
+
+**Internal changes**
+
+* The broker token and IPsec PSK are now stored in secrets which are used in preference to the corresponding fields in the Submariner CR
+which are now deprecated. For backwards compatibility and to simplify upgrades, the deprecated fields are still populated but will be removed in 0.13.
+* Submariner Globalnet no longer uses Kubeproxy chains for supporting GlobalIngressIPs. For every exported service, it now creates an
+internal ClusterIP Service with externalIP and sets the externalIPs to the globalIP assigned to the respective Service. Some deployments of
+Kubernetes do not allow the Service with externalIPs to be created for security reasons. To mitigate this, the pre-requisites mentioned in
+the Globalnet page need to be configured.
+
+**Known Issues:**
+
+* [When using the dot character in the cluster name, service discovery doesn’t work](https://github.com/submariner-io/lighthouse/issues/707).
+* On OpenShift, Globalnet metrics do not appear automatically. This can be fixed by manually opening the Globalnet metrics port, TCP/8081.
+* [When using `subctl cloud prepare` on Red Hat OpenStack Platform (RHOS), if dedicated gateway is used, security groups are associated with
+wrong node](https://github.com/submariner-io/cloud-prepare/issues/227).
+* With Red Hat OpenStack Platform (RHOS), when using a dedicated gateway node the submariner gateway security group and submariner internal
```

Comment:
@vthapar is the above point and this one same? If so, can we merge them with appropriate description.